build.sh

```
@@ -0,0 +1,73 @@
+    rm -rf /rimworld/1.6/Mods/${MOD}
+    rm -rf /rimworld/1.6-steam/Mods/${MOD}
+
+    cp -af /rimworld/1.2/Mods/${MOD} /rimworld/1.3/Mods
+    cp -af /rimworld/1.2/Mods/${MOD} /rimworld/1.4/Mods
+    cp -af /rimworld/1.2/Mods/${MOD} /rimworld/1.5/Mods
+    cp -af /rimworld/1.2/Mods/${MOD} /rimworld/1.6/Mods
+    cp -af /rimworld/1.2/Mods/${MOD} /rimworld/1.6-steam/Mods
+}
+
+function build() {
+    rm -rf /rimworld/1.2/Mods/${MOD}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)